projects/gloo/api/v1/enterprise/options/extauth/v1/extauth.proto

```diff
@@ -1613,3 +1614,17 @@ message PassThroughHttp {
     // Timeout for the auth server to respond. Defaults to 5s
     google.protobuf.Duration connection_timeout = 8;
 }
+
+// PortalAuth is used to authorize requests for credentials generated by the portal web server.
+message PortalAuth {
+    // The portal web server url used to validate credentials generated by the portal for the backing service(s).
+    string url = 1;
+    // The api key header name used to find the api key in the request headers.
+    string api_key_header = 2;
```

Comment:
Can we document somewhere, either here or elsewhere that we use API Key auth if this header is set, otherwise we default to Bearer token auth? I want to make sure this is as self documenting as possible and coming at it with a fresh perspective it isn't obvious how OAuth2 auth works.

Reply:
good call out will add a doc comment around that